ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/PrivateTransactionHandler.java

```diff
@@ -94,8 +94,7 @@ public String sendToOrion(final PrivateTransaction privateTransaction) throws Ex
     }
   }
 
-  public String getPrivacyGroup(final String key, final PrivateTransaction privateTransaction)
-      throws Exception {
+  public String getPrivacyGroup(final String key, final PrivateTransaction privateTransaction) {
```

Comment:
why did you took `Exception` out? Is this method only throwing runtime exceptions?

Reply:
Yep!

Reply:
later on we are catching a blanket `Exception`, should we change that to `RuntimeException` instead?

Reply:
Done